Makefile

```diff
@@ -22,6 +22,10 @@ ifneq ($(FDB_ENABLED), )
 	BUILD_TAGS+=foundationdb
 endif
 
+ifneq ($(MINIMAL), )
+	BUILD_TAGS+=minimal
+endif
```

Comment:
nit: I wonder if could rename `MINIMAL` to `BUILD_MINIMAL` to follow the pattern set by `BUILD_TAGS`. Also it would be great to pull the variable so that it can be tab completed from the shell.

```suggestion
# Set BUILD_MINIMAL to a non-empty value to build a minimal version of Vault with only core features.
BUILD_MINIMAL ?=
ifneq ($(strip $(BUILD_MINIMAL)),)
	BUILD_TAGS+=minimal
endif
```

Reply:
Thank you!